### PR TITLE
Docs: add some more instructions for contributing

### PR DESCRIPTION
**Pull request by philschatz**
Here's the additions. Did I miss or get something wrong @openstax/tutor-fe ?

---

# Creating a Pull Request or Issue

- include a screenshot in the **Issue/PR body**
- add either `Needs Code` or `Needs Review` label
- if it is waiting on input from someone add the `blocked` label
- if it is linked to another Issue or Pull Request, link the Issue/PR number by editing the Issue/PR **body**

### Creating a hotfix

1. Find the commit SHA to base the hotfix on
2. create a `hotfix-*` branch
3. make a Pull request to `master` (note: it may have merge conflicts)
4. add the `HOTFIX` label
5. deploy the hotfix
6. merge master in (**DO NOT REBASE!** It is important to make sure the deployed commit is in master for undoability)
7. merge the Pull request

# Code to Include

- Add selenium code to be able to exercise the feature or bug in the Pull Request
  - test by [Running Selenium](./test-integration/)
- Fix lint warnings by running `npm run lint-cjsx`
- Every element should have a `className`
  - prefix the classname with `-` if it does not appear in CSS

# Reviewing a Pull Request

- Use :+1: reaction when an issue is resolved but the comment is still visible in the Pull Request page

# Merging or Closing a Pull Request

- delete the branch

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,40 @@
 # :+1::tada: Thanks for Contributing! :tada::+1:
 
 See [openstax/CONTRIBUTING.md](https://github.com/openstax/napkin-notes/CONTRIBUTING.md) for more information!
+
+
+# Creating a Pull Request or Issue
+
+- include a screenshot in the **Issue/PR body**
```
Comment (pandafulmanda) [inline CONTRIBUTING.md]: should this include a note about, "If there are UI changes"?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
+7. merge the Pull request
+
+
+# Code to Include
+
+- Add selenium code to be able to exercise the feature or bug in the Pull Request
+  - test by [Running Selenium](./test-integration/)
+- Fix lint warnings by running `npm run lint-cjsx`
+- Every element should have a `className`
+  - prefix the classname with `-` if it does not appear in CSS
```
Comment (pandafulmanda) [inline CONTRIBUTING.md]: Should there be a a note about adding a unit test if the update is model/logic based?

**Comment by jivewise** [comment]
Should we have a note in here about having a link to the pivotal ticket if there is one?  Also 👍  on adding a moar-tests label for feedback :)

**Comment by philschatz** [comment]
Ah yep. I'll add the link-to-ticket bit. I like the idea behind "needs-more-tests" but I'm a little worried about too-many labels. Maybe we could revisit after sprint?

**Pull request merged**